src/app/fillResultsModal.js

```diff
@@ -0,0 +1,31 @@
+export const fillResultsModal = (gameHandlerResults, mode) => {
+
+    const resultsDescription = document.querySelector('#resultsDescription');
+    resultsDescription.textContent = `Congratulations ${gameHandlerResults.name}! During ${gameHandlerResults.time} seconds you answered ${gameHandlerResults.answers.length} questions and scored ${gameHandlerResults.score} point(s)!`
+
+    const resultsTable = document.querySelector('#tableWithResults table');
+
+    for (let questionItem of gameHandlerResults.answers) {
+        
+        const tableCell = document.createElement('tr');
+        tableCell.className = 'tableWithResultsQA';
+        
+        if (mode.name === 'WHAT_DOES_THIS_POKEMON_LOOK_LIKE') {
```

Comment:
It would be better to reuse contants from `modes` file (like in PokemonService) - by this way you do not risk making a typo in string literal :)

`mode === WHO_IS_THAT_POKEMON`

Reply:
Think about: "what will happen if somebody changes literals and mode.name will return different, but really similar string?" 

Then this code would not work :/